src/Controller/FileVariantController.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace GoApptiv\FileManagement\Controller;
+
+use Exception;
+use Illuminate\Http\Request;
+use GoApptiv\FileManagement\Services\FileManagement\FileManagementService;
+use Illuminate\Routing\Controller;
+use Illuminate\Support\Facades\Log;
+use GoApptiv\FileManagement\Constants;
+use GoApptiv\FileManagement\Requests\FileVariantRequest;
+use GoApptiv\FileManagement\Events\ConfirmFileVariantEvent;
+
+class FileVariantController extends Controller
+{
+    /** @var FileManagementService */
+    private $fileManagementService;
+
+    /**
+     * Constructor.
+     *
+     * @param FileManagementService $fileManagementService
+     */
+    public function __construct(FileManagementService $fileManagementService)
+    {
+        $this->fileManagementService = $fileManagementService;
+    }
+
+    /**
+     * Update Variant Id by UUID
+     *
+     * @param Request $request
+     *
+     * @return Object
+     *
+     */
+    public function updateVariant(FileVariantRequest $request)
+    {
+        try {
+            $requestData = $request->getData();
+            $uuid = $requestData->getUuid();
+            $variantId = $requestData->getVariantId();
+            $status = $requestData->getStatus();
+
+            Log::info("Updating variant id for uuid: " .$uuid);
```

Comment:
This function updates the status or variant id?

Reply:
Log message suggests it updates the variantId

Reply:
This function should update the status of the variant

Reply:
It's updating both. The Log message was not proper, updated it.